src/FlaUI.Core/Logging/TraceLogger.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace FlaUI.Core.Logging
+{
+    public class TraceLogger : AbstractBaseLogger
+    {
+        public override void GatedDebug(string message)
+        {
+            System.Diagnostics.Trace.WriteLine(message);
```

Comment:
But if you did want to add a prefix, the abstract base is probably going to be the best place to do that.